partcad-cli/src/partcad_cli/click/commands/render.py

```
@@ -34,13 +34,7 @@
         [
             "readme",
             "svg",
-            "png",
-            "step",
-            "stl",
-            "3mf",
-            "threejs",
-            "obj",
-            "gltf",
+            "png"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)